plan/newplanbuilder.go

```diff
@@ -588,14 +724,18 @@ func (b *planBuilder) buildNewTableScanPlan(tn *ast.TableName) Plan {
 	return p
 }
 
-func (b *planBuilder) buildApply(p, innter Plan, schema expression.Schema) Plan {
+func (b *planBuilder) buildApply(p, inner Plan, schema expression.Schema) Plan {
 	ap := &Apply{
-		InnerPlan:   innter,
+		InnerPlan:   inner,
 		OuterSchema: schema,
 	}
 	ap.id = b.allocID(ap)
 	addChild(ap, p)
-	ap.SetSchema(append(p.GetSchema().DeepCopy(), innter.GetSchema().DeepCopy()...))
+	innerSchema := inner.GetSchema().DeepCopy()
+	for _, col := range innerSchema {
+		col.Auxiliary = true
```

Comment:
select * from t, (select * from t1 where t1.c1 > t.c1); Is this a valid sql? If it is valid, the inner Schema is not auxiliary.